src/components/common/ProductBox/ProductBox.module.scss

```diff
@@ -16,6 +16,16 @@
     }
     overflow: hidden;
     align-items: flex-end;
+
+    .image {
```

Comment:
To nie ten task. Nowy branch róbmy z mastera, pisałem o tym na grupie.

Reply:
pierdizeli mi sie to z tymi branchami